lib/vk/schema/model/widgets_widget_page.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require 'vk/schema/model'
+
+module Vk
+  class Schema
+    # @see https://github.com/VKCOM/vk-api-schema/blob/master/objects.json
+    class Model::WidgetsWidgetPage < Vk::Schema::Model
+      # @return [Integer] Page ID
+      attribute :id, Types::Coercible::Int
+      # @return [String] Page title
+      attribute :title, Types::Coercible::String
+      # @return [String] Page description
+      attribute :description, Types::Coercible::String
+      # @return [String] URL of the preview image
+      attribute :photo, Types::Coercible::String
+      # @return [String] Page abcolute URL
+      attribute :url, Types::Coercible::String
+      # @return [Object] @see https://github.com/VKCOM/vk-api-schema/blob/master/objects.json
+      attribute :likes, Types::Coercible::Hash
+      # @return [Object] @see https://github.com/VKCOM/vk-api-schema/blob/master/objects.json
+      attribute :comments, Types::Coercible::Hash
+      # @return [Integer] Date when widgets on the page has been initialized firstly in Unixtime
```

Comment:
@houndci-bot, should you respect RuboCop configuration or should I switch you off completely because you cannot work as expected?